### DISCUSSION TITLE
Error since version 0.53.0-rc20

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#670] [General]
Hi,

I got an error in the console. See my screenshot.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/343c15d4-e2ec-4969-bd68-8693edbd305e)

**Comment by rejetto** [comment]
1. would you say you never saw this before latest update?
2. any idea of WHEN does it happen? i mean, what is causing it.
4. how often do you get it?

i think it's harmless, and i cannot exclude it's a plugin.

**Reply by PH5HP** [reply]
Hi,

Yesterday evening it was a warning of somekind. Never have seen both before. very strange. I will start the server again amd I will keep you posted.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
you may have a look at your log to see what happened at the time? <img width="115" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/4a518474-bc8c-4c38-9b0a-da1d6704b97a">

**Reply by PH5HP** [reply]
Hi,

Acces log at that time:

![image](https://github.com/rejetto/hfs/assets/140949730/84b65eb3-9471-4504-9105-c3dc846ec8d6)

![image](https://github.com/rejetto/hfs/assets/140949730/803d8597-8988-448b-959c-8b0374b6d259)

Kind regards,
Hek - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
ok thanks.
it should be something related to an upload.
i already suspected it, and i've found confirmation in my logs (i've found that error too).

**Reply by PH5HP** [reply]
Hi,

Ok ... The warning is back. The one which I saw yesterday. Not related to what you found I think.

![image](https://github.com/rejetto/hfs/assets/140949730/e63f8403-29de-4911-b032-d5279efe6c61)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by PH5HP** [reply]
Hi,

Started HFS with '--trace-warning':

![image](https://github.com/rejetto/hfs/assets/140949730/a4c30d69-4157-423a-9847-020810528ef2)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
> Ok ... The warning is back. The one which I saw yesterday. Not related to what you found I think.

what makes you think it's not related to uploads? you know there was no uploading at that time? (20:35:06)

thanks for the trace-warning, a good attempt, but sadly the result is not helping me.

**Reply by PH5HP** [reply]
Hi,

Ok. I thought it was not related when I looked at the logs, but I am no expert. What do you want me to do? At the moment the server is running for several hours now and no errors in the console.

I will gladly help you if you tell me what you want me to do. :-)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
nothing for now, thanks.
i'm often able to reproduce it.
it happens when you upload folders (many files).
i may just mute it after i'm sure it's nothing to be concerned about.

**Comment by rejetto** [comment]
ok, i've found that this was introduced by the update of node (internally to the exe)  done with latest hfs, and it goes away if go back to node 18.5.0 (but i don't want to), and that it is indeed issued by the upload, but i've verified that files are correctly written, so i'm proceeding muting it as i had already guessed.

would you care to confirm that this version works fine? it includes a few more fixes. thanks!
https://github.com/user-attachments/files/16117573/hfs-windows-x64-0.53.0-rc20.2.zip

**Reply by PH5HP** [reply]
Hi,

Thank you. I still have the warning.

![image](https://github.com/rejetto/hfs/assets/140949730/3ee17687-4408-4816-8789-53ceb7cf9185)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
this doesn't seem to be the same, i fixed the PREMATURE message.
i see you had this other message also before.

**Comment by rejetto** [comment]
ok, let's address the clientError message :)
would you say it appeared only in the last week?

**Reply by PH5HP** [reply]
Hi,

Yes. Before that I had no errors.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
any idea of when that happens?
i still have to find a way to reproduce it

**Reply by PH5HP** [reply]
Hi,

I have no idea yet. I am running HFS now in DEBUG mode. Maybe if it occurs I have some more information.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
you can look in the logs, what was happening at that time

**Reply by PH5HP** [reply]
Hi,

Acces log:
![image](https://github.com/rejetto/hfs/assets/140949730/1f4b8f71-9ada-44fe-88ce-5ba7ccf5aec6)

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
did you also look in error logs?

**Reply by PH5HP** [reply]
Hi,

Yes, but nothing to look at in that time-frame.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
ok, let's try to revert back to previous node version then.
https://github.com/user-attachments/files/16120948/hfs-windows-x64-0.53.0-rc20.3.zip

if this works, i'll keep it, to close 0.53, and postpone the problem.
let me know, and thanks a lot

**Reply by PH5HP** [reply]
Hi,

Running for several hours now with rc20,3 No errors yet.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by PH5HP** [reply]
Hi,

Still no errors.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
thank you very much

**Reply by rejetto** [reply]
released as new version, including another fix on uploads

**Comment by rejetto** [comment]
> Acces log: ![image](https://private-user-images.githubusercontent.com/140949730/346349417-1f4b8f71-9ada-44fe-88ce-5ba7ccf5aec6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyMTQ3MTcsIm5iZiI6MTcyNzIxNDQxNywicGF0aCI6Ii8xNDA5NDk3MzAvMzQ2MzQ5NDE3LTFmNGI4ZjcxLTlhZGEtNDRmZS04OGNlLTViYTdjY2Y1YWVjNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNFQyMTQ2NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODU4MjM4YmE5NWQzOWZiNjllZGEwYWI4MzRhZjYwZjVlODMxNTdhMzNkZThjODQ5MzVkZGY3ZjMyZTI2ZTM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.vtbGnaURXxWlBoPbGICR9b0oquCC8xlhWe08H2cSC7Y)

I looked again at your logs, and noticed that at those seconds you got the warning, you were being scanned by this "censys" company.
I don't know them, but I guess they mean no harm. It is still possible that they conduct some kind of study on servers that includes some unusual network interaction, leading to that warning.
I'd be glad to be scanned to see if I can reproduce the problem (although "problem" may be an exaggeration, it's probably just an annoying message).
